code/modules/projectiles/projectile/magic/abnormality.dm

```diff
@@ -75,13 +72,19 @@
 			H.gib()
 			new /mob/living/simple_animal/hostile/slime(T)
 			return BULLET_ACT_HIT
-		H.apply_damage(rand(mindmg,maxdmg), BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE))
+		H.apply_damage(damage, BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE))
```

Comment:
Does it already do this inherently, causing this to do double black damage?

Reply:
I am actually not sure, since there's a return in code that complicates everything.

Reply:
You would not be believe the rabbit hole this dragged me down. #1100

Reply:
Deals double damage. Bullet damage code is handled... elsewhere. Let me know if you want to count this as "intentional" or not.